### DISCUSSION TITLE
病毒事件波及Issue/PR收集

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#4626] [讨论]
如果你的账户被这次病毒事件波及，并被 GitHub 标记（Flagged）导致你的 Issue/PR 消失了
那些 Issue/PR 只能等账户被取消标记才会显示出来
找一个能用的账户到这下面回一贴，说明一下 **[谁]** 的 **[什么 Issue/PR]** **[编号]** 消失了
避免重复反馈无法查重或重复 PR

**Comment by MoYuan-CN** [comment]
已知信息
youzi-2333 的 导出整合包功能 (PR #4206)
消失了

**Comment by youzi-3222** [comment]
嗯，是的。我的导出整合包没了。

另外应该就是我的那个导出启动器设置 #4069 ……

我应该不需要重新开 PR 吧……反正原计划是先合资源包下载，导出启动器设置本来也计划推迟

<details>

假设有个用户，用 GitHub 的时候碰到了病毒，看到那个 mediafire 的链接，所以用自己的代码时间来提交一个 comment，但因为不懂 GitHub 有多傻逼，提醒了非常多的用户，并且还在病毒发出来的一瞬间就提交了。

那么，我们应该因为他提醒别人但是极有可能是机器人，就阴阳怪气地把他 flag 了吗？

普通用户当然不懂 GitHub 有多傻逼。他们甚至还要为了提醒别人去全 GitHub 搜索，乃至折腾文案怎么写最好。我们应该先假定对方是善意的，理解对方的付出，并保持基本的尊重。

这就是为什么这个态度绝对不可取（甚至是在确实他提醒了，并且这个 comment 写的一点毛病都没有的情况下）：

![f1d82fdcf566b884865baa9ef7c5f2a2](https://github.com/user-attachments/assets/a8bed911-b3c1-49bf-ade8-818272489651)

> 那么我想说，社区的定义是什么？完全与别人的 Repo 一个 Issue 都没有也能说是社区？

**能。**

如果你使用 GitHub，并且还搜到了这个 issue，那你自然是 GitHub 社区中的一员，那么你当然也有资格，对社区中傻逼 GitHub 的 flag 问题做出评价。

这类几乎没有违反过社区准则，也不懂 GitHub 有多傻逼的用户，正是 GitHub 最容易伤害到的用户——我觉得他们甚至比 GitHub 更有发言权。

不如换个角度，为什么你认为不能呢？

这个说法更像是 ”没有参与贡献的普通用户 ≠ 社区“，或者更直白一点，”有贡献的用户 > 没贡献的用户“，这是绝对错误的。

GitHub 存在 ”管理团队“，不平等看待不同水平和贡献量的用户。你有没有贡献、有没有权限，在他那里影响很多事情。

考虑到 wuyuncheng-26 已经多次提醒了注意封禁问题，但这之后还发生了上面截图的 youzi-2333，在正式警告之后还有 wuliaodexiaoluo……

编不下去了。

</details>

**Reply by MoYuan-CN** [reply]
重启 PR #4206 先，谢啦

**Comment by tangge233** [comment]
一年又一个月了，GitHub Support 那有什么动静么👀

**Reply by LinQingYuu** [reply]
直接 Reject 了说是（
